fundamentals/examples.tex

```diff
@@ -151,79 +151,111 @@ \subsection{Simulations}
+is sent there if possible.  All these preferences are easy to adjust and
+\Cyclus will dynamically handle supply constraints and non-uniform preference
+resolution.  It is notable that separations and recycle fuel fabrication
+capacity is deployed identically in all simulations.  In the once-through
+case, the recycling loop never acquires material, and so reactors always
+receive fresh UOX fuel.  In the modified open case, the recycle loop only
+acquires spent UOX.  The \gls{DRE} ensures everything operates smoothly in all
+cases.
+
+Cyclus' discrete materials make single-pass recycle easy to implement.  The
+reactors keep track of fuel as discrete batches. A reactor remembers where it
+received each batch from.  If a batch was received from the recycled fuel
+fabrication facility, it does not offer it to separations and instead sends it
+to the repository directly.  The material flows for the modified open case are
+shown in Figure \ref{fig:flowmodopen}.
 
 \begin{figure}[H]
 \begin{center}
 \includegraphics{./images/flow-mod-open-1.eps}
 \end{center}
-\caption{Modified open 1-pass \gls{MOX} recycle fuel cycle material flows.}
+\caption{Modified open 1-pass \gls{MOX} recycle scenario material flows.}
 \label{fig:flowmodopen}
 \end{figure}
 
-Changing this to a fully closed fuel cycle entails only a one-word change in the input file
-regarding the output commodity for the spent \gls{MOX} fuel of the
-\Class{BatchReactor}: 
+Changing the scenario from a modified open fuel cycle to a closed fuel cycle
+requires only a one-word change in the input file regarding the output
+commodity for the spent \gls{MOX} fuel of the \Class{Reactor}: 
 
 \begin{lstlisting}[language=diff]
-  --- mod-open-1.xml	2014-10-08 08:31:33.892523173 -0500
-  +++ closed-1.xml	2014-10-08 08:31:33.892523173 -0500
-  @@ -108,7 +108,7 @@
-           <fuel>         
-             <incommodity>mox_fuel</incommodity>
-             <inrecipe>lwr_mox_fuel</inrecipe>
-  -          <outcommodity>spent_mox</outcommodity>
-  +          <outcommodity>spent_fuel</outcommodity>
-             <outrecipe>mox_spent_fuel</outrecipe>
-           </fuel>
+     <fuel>         
+       <incommodity>mox</incommodity>
+  -    <outcommodity>waste</outcommodity>
+  +    <outcommodity>spent_fuel</outcommodity>
+       <inrecipe>mox_resh_fuel</inrecipe>
+       <outrecipe>mox_spent_fuel</outrecipe>
+     </fuel>
 \end{lstlisting}
 
-This results in the material flows in Figure \ref{fig:flowclosed}. Note
-that the \Class{BatchReactor} always transmutes fuel into the same
-composition, so the nuclide level flows are unrealistic for the full, 
-multi-pass
-recycle case.
+This results in the material flows in Figure \ref{fig:flowclosed}.  A
+similarly trivial change was used to switch from the once through to a
+modified open fuel cycle.  Note that the \Class{Reactor} always transmutes
+fuel into fixed compositions, so nuclide-specific flows become
+somewhat unrealistic in the fully closed cycle case.
```

Comment:
When you say something like "somewhat unrealistic" in a paper, you then have to dedicate an entire section to what is unrealistic, why it's unrealistic, and what you plan to do about it. Suggestion: rather than "nuclide-specific flows... " try "isotopic compositions are approximate rather than exact in the fully closed cycle case."